Description: Add tflite target to tensorflow/BUILD
 <long description that can span multiple lines, optional>
Author: Francis Murtagh <francis.murtagh@arm.com>
Origin: <upstream|backport|vendor|other>, <URL, required except if Author is present>
Bug: <URL to the upstream bug report if any, implies patch has been forwarded, optional>
Bug-<Vendor>: <URL to the vendor bug report if any, optional>
Forwarded: <URL|no|not-needed, useless if you have a Bug field, optional>
Applied-Upstream: <version|URL|commit, identifies patches merged upstream, optional>
Reviewed-by: <name and email of a reviewer, optional>
Last-Update: 2021-06-02 <YYYY-MM-DD, last update of the meta-information, optional>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/BUILD
+++ b/BUILD
@@ -6,3 +6,12 @@ exports_files(
         "configure.py",
     ],
 )
+
+cc_binary(
+         name = "libtensorflow_lite_all.so",
+         linkshared = 1,
+         deps = [
+                 "//tensorflow/lite:framework",
+                 "//tensorflow/lite/kernels:builtin_ops",
+         ],
+ )
